### ISSUE TITLE
DOSBox-Optionals two suggestions

### ISSUE THREAD

**Issue by postalesdeamor** [MartyShepard/DOSBox-Optionals#18]
Add option to invert joystick axis, since some games require it.

In general, improve the 3Dfx implementation (a lot of games fail totally or partially). Perhaps it would be enough to update the glide2x.ovl library.

Greetings and thanks :)

**Comment by MartyShepard** [comment]
Hello, Invert Axis can i add.

3DFX Code need more Free Time :)

#5

**Comment by postalesdeamor** [comment]
Of course, mate. There is no rush and there are many of us who appreciate your work on DosBox Optionals.

I have been implementing HyperSpin under RockeLauncher for several years and have chosen your build for the MS-DOS system.

I am selecting which ones I keep in my collection, from a selection of 7,500 games. I generally follow an alphabetical order and only a few games, which have given me problems with your build, I am launching from other builds. 

![dosbox](https://user-images.githubusercontent.com/96334148/153555850-a25bc3bc-e098-4cb8-baae-48085ac1c809.jpg)

Greetings and thanks for your dedication.

**Comment by MartyShepard** [comment]
InvertY for 3 Axsis in the Upcoming version added.

**Comment by postalesdeamor** [comment]
Great!

Thanks a lot for attend my request.
